backend/Api/Validators/ConsultantValidators.cs

```diff
@@ -0,0 +1,43 @@
+using System.Net.Mail;
+using Api.Routes;
+using Core.DomainModels;
+
+namespace Api.Validators;
+
+public static class ConsultantValidators
+{
+    public static bool IsValidEmail(string email)
+    {
+        var valid = true;
+
+        try
+        {
+            var mailAddress = new MailAddress(email);
+        }
+        catch
+        {
+            valid = false;
+        }
+
+        return valid;
```

Comment:
```suggestion
return MailAdress.TryCreate(email, out_)
```
This will return true if it could parse the email. Similar to `TryParse<>(json, out _)`

It's common to use the "_" name for unused out variables